pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -785,3 +792,24 @@ func (c *Cluster) getOSDLabels(osdID int, failureDomainValue string, portable bo
 		portableKey:         strconv.FormatBool(portable),
 	}
 }
+
+//GetVolumeGroupName returns the Volume group name from the given Logical Volume Path
```

Comment:
Looks like this method is only called by the `daemon` package. How about if you move this method to that package?

Reply:
moved both `getVolumeGroupName` and `releaseLVMDevice` to `daemon` package as they are only used there